src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -260,6 +263,30 @@ public void writeHeader(final VariantContextWriter vcfWriter, final Set<VCFHeade
         }
     }
 
+    private void removeReadStubs(final AssemblyRegion assemblyRegion) {
+        final List<GATKRead> readStubs = assemblyRegion.getReads().stream()
+                .filter(r -> r.getLength() < AssemblyBasedCallerUtils.MINIMUM_READ_LENGTH_AFTER_TRIMMING).collect(Collectors.toList());
+        assemblyRegion.removeAll(readStubs);
+    }
+
+    private void removeUnmarkedDuplicates(final AssemblyRegion assemblyRegion) {
```

Comment:
This looks like it considers:

- SVs:  This code checks whether the mates are clustered, even if on a different contig from the supporting reads of the variant.
- Duplicate start and end positions of supporting reads.

Please tell me if I have misinterpreted.  If I have not misinterpreted, then no action required.

Reply:
That's right.